package.json

```
@@ -80,6 +80,7 @@
     "react-scripts": "5.0.1",
     "source-map-explorer": "2.5.3",
     "typescript": "4.9.5",
+    "typescript-eslint": "^7.8.0",
```

nit: please pin the version explicitly
```suggestion
    "typescript-eslint": "7.8.0",
```